app/containers/ItunesAppContainer/saga.js

```diff
@@ -0,0 +1,16 @@
+import { takeLatest, call,put } from 'redux-saga/effects';
+import { itunesAppContainerTypes, itunesAppContainerCreators } from './reducer';
+import { getSongsApi } from '@services/ituneSongsApi';
+const { REQUEST_SEARCH_SONG } = itunesAppContainerTypes;
+const {setSongs,setLoader}=itunesAppContainerCreators
+
+export function* requestSearchSong(action) {
+  yield put(setLoader(true))
+  const songs=yield call(getSongsApi, action.artistName);
```

Comment:
let's handle the failure case as well

Reply:
apisauce wraps adds an `ok` property to the response, which can be used to determine whether the api call was successful or not

Reply:
we can also add try catch block like
```

try{
  const songs=yield call(getSongsApi, action.artistName);

}catch(err){

}
```